lib/tree.js

```diff
@@ -1,10 +1,51 @@
 const spawn = require('child_process').spawn;
 
-module.exports = function(rootPid, callback) {
+function find_kids_os390(rootPid, callback) {
```

Comment:
Please rename to remove "kids" from this code. My own code policy after losing my own child is not to refer to children, kids, etc when meaning processes (this does appear in old code, but I'm making sure new code doesn't come in with it).

You'll also need to rebase your code from the latest PR - which isn't a huge change.